frontend/dockerfile/docs/reference.md

```diff
@@ -1135,33 +1140,22 @@ RUN apt-get update && apt-get install -y ...
 
 ## ADD
 
-ADD has two forms:
+ADD has two forms.
+The latter form is required for paths containing whitespace.
 
 ```dockerfile
-ADD [--chown=<user>:<group>] [--chmod=<perms> [--exclude=<exclude>]... [--checksum=<checksum>] <src>... <dest>
-ADD [--chown=<user>:<group>] [--chmod=<perms> [--exclude=<exclude>]... ["<src>",... "<dest>"]
+ADD [OPTIONS] <src> ... <dest>
+ADD [OPTIONS] ["<src>", ... "<dest>"]
```

Comment:
Note to reviewer: IMO this instruction has too many flags now to show them as a one-liner, so I moved all the flags to a list below the synopsis, with links